static/src/javascripts/projects/common/modules/experiments/ab-test-clash.js

```diff
@@ -13,8 +13,13 @@ define([
             variants: ['control', 'member-contribute']
         };
 
+        var contributionsMembershipEpicSideBySide = {
+            name: 'ContributionsMembershipEpicSideBySide',
+            variants: ['control', 'member-contribute']
```

Comment:
yes it does. Both are variants where membership and contribute are displayed. the difference is here they are both buttons rather than one buttons and one text. But member-contribute still makes sense